### PR TITLE
DDF-3017 Update security-encryption-commands to use new Karaf command annotations

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Update security-encryption-commands to use new Karaf command annotations
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@vinamartin @mcalcote @emmberk 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@stustison 

#### How should this be tested? (List steps with links to updated documentation)
Verify build passes and security:encrypt command works
#### What are the relevant tickets?
[DDF-3017](https://codice.atlassian.net/browse/DDF-3017)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by clockard** [comment]
build now

File: platform/security/encryption/platform-security-encryption-commands/pom.xml [hunk 1]
```diff
@@ -34,13 +34,25 @@
...
-            <artifactId>org.apache.karaf.shell.console</artifactId>
+            <artifactId>org.apache.karaf.shell.core</artifactId>
             <version>${karaf.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.osgi</groupId>
-            <artifactId>org.osgi.core</artifactId>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${org.slf4j.version}</version>
```
Comment (mcalcote) [inline platform/security/encryption/platform-security-encryption-commands/pom.xml]: I don't see this used in the change set. Do we need this added here?

Comment (mcalcote) [inline platform/security/encryption/platform-security-encryption-commands/pom.xml]: On a closer look, I see it is used in the test class. Should we add scope to this dependency?

**Comment by josephthweatt** [comment]
I also pulled your code and verified it works

**Comment by clockard** [comment]
build now

File: platform/security/encryption/platform-security-encryption-commands/pom.xml (hunk 1 above)
Comment (clockard) [inline platform/security/encryption/platform-security-encryption-commands/pom.xml]: Will add test scope to this dependency

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**